# This file is part of HiGFXback

# requires
REQUIRES="autotools-wrappers-lt-build"

if ! pkg-config --exists --print-errors $REQUIRES; then exit 1; fi

# configure (ac-2.69; am-1.16; lt-2.4.6)
libtoolize -c; aclocal; autoconf; autoheader -f; automake -a -c
./configure --disable-static --prefix=

# build
make

# install
install -d $DESTDIR/include
install -m 644 pixman/pixman.h $DESTDIR/include
install -m 644 pixman/pixman-version.h $DESTDIR/include
install -d $DESTDIR/lib
install pixman/.libs/libpixman-1.so.0.* $DESTDIR/lib/libpixman-1.so.0
ln -sf libpixman-1.so.0 $DESTDIR/lib/libpixman-1.so
install -d $DESTDIR/lib/pkgconfig
install -m 644 pixman-1.pc $DESTDIR/lib/pkgconfig

# build.pc
install -d $DESTDIR/share/pkgconfig
cat > $DESTDIR/share/pkgconfig/pixman-build.pc << EOF
Name: pixman
Version: 0.36.0
Description: pixel manipulation library
Requires: $REQUIRES

devel=\\
/include/pixman.h \\
/include/pixman-version.h \\
/lib/libpixman-1.so \\
/lib/pkgconfig/pixman-1.pc

exec=\\
/lib/libpixman-1.so.0
EOF

exit
--- pixman-0.36.0.orig/configure.ac
+++ pixman-0.36.0/configure.ac
@@ -59,6 +59,7 @@
 m4_define([pixman_version],[pixman_major.pixman_minor.pixman_micro])
 
 AC_INIT(pixman, pixman_version, [pixman@lists.freedesktop.org], pixman)
+AC_CONFIG_MACRO_DIR([m4])
 AM_INIT_AUTOMAKE([foreign dist-bzip2])
 
 # Suppress verbose compile lines
--- pixman-0.36.0.orig/demos/Makefile.am
+++ pixman-0.36.0/demos/Makefile.am
@@ -1,3 +1,5 @@
+AUTOMAKE_OPTIONS = subdir-objects
+
 EXTRA_DIST = parrot.c parrot.jpg scale.ui
 
 if HAVE_GTK
